public/styles.css

```diff
@@ -13,51 +14,12 @@ div h1 {
   color: #a35d6a;
   text-align: center;
 }
-h2{
+h2 {
   color: #a35d6a;
   text-align: center;
 }
-.2ndhalf{
```

Comment:
moving styles to its own component files, e.g. `.srch` was moved to the `Search.css`